server/graphql/v2/query/collection/AccountsCollectionQuery.ts

```diff
@@ -120,10 +125,16 @@ const AccountsCollectionQuery = {
     } else {
       const cleanTerm = args.searchTerm?.trim();
 
+      let hostCollectiveIds;
+      if (args.host) {
+        const hosts = await fetchAccountsWithReferences(args.host);
+        hostCollectiveIds = hosts.map(host => host.id);
+      }
+
```

Comment:
Cheap optimizations:

```suggestion
      const hostCollectiveIds = await fetchAccountsIdsWithReference(args.host);
```